Assets/Scripts/SS3D/Systems/Inventory/Items/Item.cs

```diff
@@ -272,11 +273,26 @@ public bool HasTrait(Trait trait)
         /// Modify the container of this item, can pass null to make this item not depending on any container.
         /// </summary>
         [Server]
-        public void SetContainer(AttachedContainer newContainer)
+        public virtual void SetContainer(AttachedContainer newContainer)
         {
-            if (_container == newContainer)
+            IEnumerable<Collider> colliders = GetComponents<Collider>().Concat(GetComponentsInChildren<Collider>());
```

Comment:
Getting the Collider via get component seems hacky, can't we get it via the AttachedContainer's container contents that are Items? and have a reference to all the colliders in the Item script?

Reply:
hmmm there's one issue I did not think of .. What about items that are container themselves ? When dropping such an item, it will enable all colliders including the ones of items contained in the dropped item. Which is unwanted behavior. we might have to keep a list of colliders for each item, but it means setting manually for each item their colliders which is not fun .. And I'd rather use a work around, unless the logic get out of hand. I think the best would be to keep getcomponentinchildren but add a condition with linq to remove all child gameobject of a gameobject with a container.